api-reference/docstubs/api/user-delta.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "delta"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# delta
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /users/delta
+GET /me/managedDevices/{managedDeviceId}/users/delta
```

Comment:
Line 32 - is this syntax supported?
Just not sure, as the live topic doesn't mention this.

Reply:
circular issue and busted.  @wandeg to investigate